introduction.md

```diff
@@ -1 +1,5 @@
-Hi I'm rafal. (m3ntos). My favourite course is of course Git, but Ruby would be the second one.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?